src/Common/tests/Performance/PerfTestBase.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.IO;
+
+namespace System
+{
+    /// <summary>Base class for performance test classes</summary>
+    public abstract class PerfTestBase
```

Comment:
I wonder if we should just separate these out into a "PerfUtils" class or something. It's possible that we might want to have a single base class for some library's Unit tests and Perf tests, and this wouldn't work well in that case. That's just speculation though. It would make the second method here more cumbersome to use than it is currently.